src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -298,7 +298,7 @@ else if (!result.getParticipation().getId().equals(participationId)) {
      * @param resultId     - the id of the result which should get deleted
      * @return 200 Ok response if canceling was successful, 403 Forbidden if current user is not an instructor of the course or an admin
      */
-    @DeleteMapping("participations/{participationId}/submissions/{submissionId}/results/{resultId}")
+    @DeleteMapping("participations/{participationId}/text-submissions/{submissionId}/delete/{resultId}")
```

Comment:
```suggestion
    @DeleteMapping("participations/{participationId}/text-submissions/{submissionId}/results/{resultId}")
```

Reply:
I don't think we should use `delete` here as part of the URL.
This should also be changed in the other cases

Reply:
I don't see those changes reflected in the server integration tests (they fail at the moment)